cmd/create/cluster/cmd.go

```diff
@@ -1140,14 +1145,17 @@ func run(cmd *cobra.Command, _ []string) {
 		if privateLink {
 			existingVPCHelp += "For PrivateLink, only a private subnet per availability zone is needed."
 		}
-		useExistingVPC, err = interactive.GetBool(interactive.Input{
-			Question: "Install into an existing VPC",
-			Help:     existingVPCHelp,
-			Default:  useExistingVPC,
-		})
-		if err != nil {
-			reporter.Errorf("Expected a valid value: %s", err)
-			os.Exit(1)
+
+		if !useExistingVPC {
```

Comment:
This check is redundant - it's already inside a block that checks `!useExistingVPC`